src/app/research/research.component.html

```diff
@@ -1,42 +1,40 @@
 <h1 class="center">Full Publication List</h1>
 
-<div>
-  <ul class="center publication-font" style="list-style: none;">
-    <li>
-      <b><u>LIBSA - A method for the determination of ligand-binding preference to allosteric sites on receptor ensembles</u></b>
+<div class="center publication-font">
+    <div>
+      <b><u><a href="http://pubs.acs.org/doi/pdf/10.1021/ci400474u">LIBSA - A method for the determination of ligand-binding preference to allosteric sites on receptor ensembles</a></u></b>
       <br/>
       <b>Jan 20, 2014  Journal of Chemical Information and Modelling</b>
-    </li>
+    </div>
     <br/>
-    <li>
-      <b><u>Isoform Selectivity of Adenylyl Cyclase Inhibitors: Characterization of Known and Novel Compounds</u></b>
+    <div>
+      <b><u><a href="http://jpet.aspetjournals.org/content/347/2/265.long">Isoform Selectivity of Adenylyl Cyclase Inhibitors: Characterization of Known and Novel Compounds</a></u></b>
       <br/>
       <b>Sep 4, 2013  The Journal of Pharmacology and Experimental Therapeutics</b>
-    </li>
+    </div>
     <br/>
-    <li>
-      <b><u>Andrographolide derivatives inhibit guanine nucleotide exchange and abrogate oncogenic Ras function</u></b>
+    <div>
+      <b><u><a href="http://www.pnas.org/content/110/25/10201.abstract">Andrographolide derivatives inhibit guanine nucleotide exchange and abrogate oncogenic Ras function</a></u></b>
       <br/>
       <b>May 8, 2013  Proceedings of the National Academy of Sciences</b>
-    </li>
+    </div>
     <br/>
-    <li>
-      <b><u>Membrane Targeting Methods</u></b>
+    <div>
+      <b><u><a href="http://onlinelibrary.wiley.com/doi/10.1002/9780470015902.a0002615.pub2/abstract?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base%3BJ4cBPv9nS%2BKCzDbj%2BePfsw%3D%3D">Membrane Targeting Methods</a></u></b>
       <br/>
       <b>Jan 15, 2012  Encyclopedia of Life Sciences</b>
-    </li>
+    </div>
     <br/>
-    <li>
-      <b><u>Novel Allosteric Sites on Ras for Lead Generation</u></b>
+    <div>
+      <b><u><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0025711">Novel Allosteric Sites on Ras for Lead Generation</a></u></b>
       <br/>
       <b>Oct 25, 2011  PLoS One</b>
-    </li>
+    </div>
     <br/>
-    <li>
-      <b><u>Comparison of Normalization Methods for the Identification of Biomarkers Using MALDI-TOF and SELDI-TOF Mass Spectra</u></b>
+    <div>
+      <b><u><a href="https://www.ncbi.nlm.nih.gov/pubmed/20141334?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base%3BJ4cBPv9nS%2BKCzDbj%2BePfsw%3D%3D">Comparison of Normalization Methods for the Identification of Biomarkers Using MALDI-TOF and SELDI-TOF Mass Spectra</a></u></b>
       <br/>
       <b>Feb 14, 2010  OMICS: A Journal of Integrative Biology</b>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <b> tag by <strong>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)